mirrord/cli/src/error.rs

```diff
@@ -400,7 +409,12 @@ impl CliError {
 
         match error {
             KubeApiError::KubeError(Error::Auth(AuthError::AuthExec(error))) => {
-                Self::KubeAuthExecFailed(error)
+                Self::KubeAuthExecFailed(error.to_owned())
+            }
+            KubeApiError::KubeError(Error::Service(ref fail))
+                if fail.to_string().contains("InvalidCertificate") =>
+            {
+                Self::InvalidCertificate(error)
```

Comment:
I'd change function name now, it's no longer just `auth_exec_error_or`. Maybe sth like `detect_known_error_or_else`? Maybe sth shorter :v